src/app/contexts/ToggleContext/reducer.jsx

```diff
@@ -0,0 +1,19 @@
+// toggle actions
+export const updateToggles = data => ({
+  type: 'UPDATE_TOGGLE_STATE',
+  data,
+});
+
+export const toggleReducer = (toggleState, action) => {
+  console.log('toggleState', toggleState);
+  const { type, data } = action;
+  console.log('data', data);
+  switch (type) {
+    case 'UPDATE_TOGGLE_STATE':
+      // TODO: In future update toggles with the data passed in, this would be from a 3rd party API.
+
+      return { ...toggleState, ...data.toggles };
```

Comment:
Should a response from the toggles service completely overwrite what we store in the app? i.e. should the toggles service be the source of truth? If so then we maybe shouldn't spread the previous toggle state and just do
```js
return { data.toggles }
````

Reply:
Once all toggles are moved to the feature toggle service instead of merging data we will simply overwrite the initial toggles state.